scripts/generate_protobuf_and_funcs.rb

```diff
@@ -441,6 +441,39 @@ def generate!
   // Named tokens in scan.l
   #{@scan_protobuf_tokens.join("\n  ")}
 }
+
+message SummaryResult {
+  message SummaryResultTable {
+    string table = 1;
+    // FIXME(duckinator): I AM NOT CONFIDENT Node IS RIGHT. THIS NEEDS TO BE VERIFIED.
+    Node context = 2;
+  }
+  // tables: Vec<(String, Context)>
+  repeated SummaryResultTable tables = 1;
+
+  // aliases: HashMap<String, String>
+  map<string, string> aliases = 2;
+
+  // cte_names: Vec<String>
+  repeated string cte_names = 3;
+
+  // functions: Vec<(String, Context)>
+  message SummaryResultFunction {
+    string name = 1;
+    // FIXME(duckinator): Just like with SummaryResultTable, I am *NOT* confident using Node is right, and this needs to be verified.
+    Node context = 2;
```

Comment:
Same question here about the protobuf equivalent of `Context`. Is that `Node`, or do I need to add something else?